nmigen/vendor/xilinx_spartan_3_6.py

```diff
@@ -437,24 +437,25 @@ def get_ff_sync(self, ff_sync):
         m.d.comb += ff_sync.o.eq(flops[-1])
         return m
 
-    def get_reset_sync(self, reset_sync):
-        if reset_sync._max_input_delay is not None:
-            raise NotImplementedError("Platform '{}' does not support constraining input delay "
-                                      "for ResetSynchronizer"
-                                      .format(type(self).__name__))
-
+    def get_async_ff_sync(self, async_ff_sync):
         m = Module()
-        m.domains += ClockDomain("reset_sync", async_reset=True, local=True)
+        m.domains += ClockDomain("async_ff", async_reset=True, local=True)
         flops = [Signal(1, name="stage{}".format(index), reset=1,
```

Comment:
Looking at this, doesn't this platform need to throw if input delay is set? Since it's not supported?

Reply:
Yep, looks like this fell through the cracks.